backend/shared/neuroDivergenceHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { app } from '../config'
+
+export const canViewNeurodivergenceSupportData = (agencyId: string, enabledPrisons): boolean => {
```

Comment:
Having got familiar with the code, it would seem that the `esweService` is the best place for this.

But it's not that important